source/reference/api/reverse.txt

```diff
@@ -99,3 +97,25 @@ entire sync process to copy the original data.
 To view the mapping direction for the synchronization of the source and
 destination clusters, use the :ref:`progress <c2c-api-progress>`
 endpoint and check the ``directionMapping`` object.
+
+.. _c2c-validate-unique-index:
+
+Validate Unique Indexes
+~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to reverse direction, ``mongosync`` requires that all
+:ref:`unique <index-type-unique>` indexes use the correct formatting. 
+Clusters of servers that began using MongoDB 4.2 or older and were since
```

Comment:
[question]

Is this meant to be "Clusters `or` servers" ?

Reply:
Fixed